{"name": "test_login", "status": "broken", "statusDetails": {"message": "NameError: name 'excel' is not defined", "trace": "self = <testcase.test_login_demo.TestLogin object at 0x000002286AD08390>\n\n    def test_login(self):\n        for i in range(1, 10):\n>           if (excel[i]['是否执行']) != 'Y':\nE           NameError: name 'excel' is not defined\n\ntest_login_demo.py:20: NameError"}, "start": 1656905270804, "stop": 1656905270804, "uuid": "72eca649-2a33-4d8c-a202-6ec65af3be68", "historyId": "966da4c978c322b6b1d09eb1e0322926", "testCaseId": "635f167e9ec99310021d4ad82ace208f", "fullName": "test_login_demo.TestLogin#test_login", "labels": [{"name": "suite", "value": "test_login_demo"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "26928-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_demo"}]}